inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -75,6 +77,58 @@ def from_text_file(
         depths = np.loadtxt(text_file_path)
         return cls(depths, x_range, y_range)
 
+    @classmethod
+    def from_random_depths(
+        cls,
+        n: int,
+        x_range,
+        y_range,
+        max_depth: float = 10,
+        initial_roughness: float = 1,
+        roughness_factor: float = 0.5,
+        percentile_above_water: float = 20,
+    ):
+        """Creates a `Bathymetry` object with random depths.
+
+        A grid of random depths of size (2^n + 1) x (2^n + 1) is created, where
```

Comment:
Maybe we want to make this easier for the user.
I think we should allow the user to directly specify a grid size (e.g. 800 x 600) and then we internally select the right power of 2 and in the end crop the result and ensure the percentile_above_water.

I think this creates a more natural interface.

Reply:
That's a quite intricate choice, relating the percentile above water and the grid size. I think in general it is much more natural that the user "gets what they ask for". 

WDYT of the following:
1. We allow users to specify a grid size (e.g. 800 x 600)
2. We take the maximum size (e.g. 800) and figure out the corresponding n (e.g. 10)
3. We use that n to produce a random bathymetry
4. We interpolate the random grid to a grid with the size selected by the user (e.g. 800 x 600)

Reply:
I think interpolation is fair and is a good solution, because this is a random bathymetry anyway so the interpolation will not "distort" anything. If we see some weird effect, then we can think about correcting it, but for now, I think this is a good solution. Please proceed with the interpolation.

Reply:
Just questioning:
- Shall we overcomplicate the code at the moment? 
- Could users just say "high", "medium", "low" resolution and we define an appropriate `n`based on that?

Reply:
I think it's ok to keep numbers instead of a choice between "high", "medium", "low" to control the size of the grid.